examples/pymc3_howto/sampling_compound_step.ipynb

```diff
@@ -12,10 +12,11 @@
     "# Compound Steps in Sampling\n",
```

Comment:
<p>I would add that comment as a markdown cell and not print the whole dataarray</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>does printing one chain work?</p>

Reply:
<p><code>trace1.sample_stats["accept"].sel(chain=1).values</code></p><p> </p><p>I'm going for this because it helps actually notice the two sampler step accept thing and this doesnt take up much space</p>